### PR TITLE
fix golint errors in pkg/routes

### PR DESCRIPTION
**Pull request by warmchang**
**What this PR does / why we need it**:
This PR fix golint errors in pkg/routes

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

/release-note-none
/kind cleanup

### CODE REVIEW

**Review by cheftako — APPROVED** [review]
/lgtm

**Comment by nikhita** [comment]
/assign @sttts 
for approval

File: pkg/routes/logs.go [hunk 1]
```diff
@@ -26,6 +26,7 @@ import (
 // Logs adds handlers for the /logs path serving log files from /var/log.
 type Logs struct{}
 
+// Install func registers the logs handler.
```
Comment (sttts) [inline pkg/routes/logs.go]: nit: leaves out "func"

**Pull request merged**